mne/epochs.py

```diff
@@ -1821,13 +1827,20 @@ def get_data(self, picks=None, item=None, units=None, tmin=None, tmax=None):
             End time of data to get in seconds.
 
             .. versionadded:: 0.24.0
+        copy : bool | None
+            If true (default) then a copy is returned. If false,
+            a view is returned if the requirements are met.
+            If picks, item, tmin or tmax are not None, a copy
+            is returned.
```

Comment:
```suggestion
            Whether to return a copy of the object's data, or (if possible) a view.
            See :std:label:`basics.copies-and-views <numpy:basics.copies-and-views>`
            for an explanation. Default is ``True``.
```

Not sure I got the cross-ref syntax right here...